### ISSUE TITLE
Severe memory leak in glDrawElements on Windows

### ISSUE THREAD

**Issue by rcdailey** [powervr-graphics/Native_SDK#31]
Not sure internally what is going on, but over time we consistently and continuously leak memory. We eventually get to the point where our application crashes because it is out of memory. If I comment out the glDrawElements function, the memory leak is gone. Unfortunately the code for your EGL libraries is not open source, so I don't have any additional information to share.

What can we do to find out what is causing the memory leak?

Using PVR version 5.2

**Comment by jamesRumbleh** [comment]
Hi Robert,

Do you happen to have a minimal reproduction case? Or alternatively could you provide us with a binary we could use to attempt to reproduce the problem on our end? As you've said our PVRVFrame libraries (lib/Windows_x86_64/libEGL*, lib/Windows_x86_64/libGLES_CM*, lib/Windows_x86_64/libGLESv2*) are closed source.

**Comment by PVRDevTech** [comment]
Hi Robert,

Do you happen to have a minimal reproduction case? Or would you be able to provide us with a binary which reproduces the problems you're seeing so that we can attempt to reproduce the problem on our end? As you've mentioned the PVRVFrame libraries in lib/Windows_x86_64/libEGL, lib/Windows_x86_64/libGLES_CM and lib/Windows_x86_64/libGLESv2 are closed source.

**Comment by rcdailey** [comment]
Sorry for taking so long to get back to you on this. Since I reported this issue, I was not able to get time to build a reproducible case. Due to these issues we eventually moved away from PVR on Windows and instead we use Google SwiftShader now. This avoids a few other driver upgrade issues we had as well. Our main platform for our product is Android, not Windows, so everything we do on Windows is stop-gap anyway. This was an easier direction.

I appreciate your time. I'll close this issue.
